src/components/VmActions/ConfirmationModal.js

```diff
@@ -30,7 +30,7 @@ const ConfirmationModal = ({ show, title, confirm, body, onClose, extra, accessi
       <Modal.Footer>
         { extra && <button className='btn btn-info' onClick={() => { extra.onClick(); onClose() }}>{extra.title}</button> }
         <button className={`btn ${confirm ? 'btn-default' : 'btn-info'}`} onClick={onClose}>{msg.cancel()}</button>
-        { confirm && <button className='btn btn-info' onClick={() => { confirm.onClick(); onClose() }}>{confirm.title}</button> }
+        { confirm && <button className={`btn ${confirm.type ? `btn-${confirm.type}` : 'btn-info'}`} onClick={() => { confirm.onClick(); onClose() }}>{confirm.title}</button> }
```

Comment:
The `confirm.type` isn't part of the `confirm` prop types.  It would be appropriate to add it...

```jsx
ConfirmationModal.propTypes = {
  show: PropsTypes.bool,
  title: PropsTypes.string.isRequired,
  onClose: PropsTypes.func,
  accessibleDescription: PropsTypes.string,
  confirm: PropsTypes.shape({
    title: PropsTypes.string,
    type: PropsTypes.oneOf([ 'primary', 'success', 'info', 'warning', 'danger' ]),
    onClick: PropsTypes.func,
  }),
  extra: PropsTypes.shape({
    title: PropsTypes.string,
    onClick: PropsTypes.func,
  }),
  body: PropsTypes.oneOfType([PropsTypes.node, PropsTypes.string]).isRequired,
}
```

...and set the list of possible values (the `PropsTypes.oneOf`) to what is expected to be used.

Reply:
@sjd78 @bond95 @yodem I wonder if this repeats #1060?

Reply:
I agree to the change, it will make the confirmationModal more generic and will give him more options.

It does repeats from #1060 and I think this solution is fit to specifically these issues but can solve more issues that regards to the type of button in the future as well.

Reply:
> @sjd78 @bond95 @yodem I wonder if this repeats #1060?
@thaorell Oops..you are right. Those 2 issues handle the same except that in #1060 the text message was also changed. I guess we'll need to unified them into one PR and close the other issue as duplicated.